app/routes/index.js

```diff
@@ -1,18 +1,26 @@
-import { inject as service } from '@ember/service';
 import Route from '@ember/routing/route';
-// eslint-disable-next-line ember/no-mixins
-import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
-import { hash } from 'rsvp';
+import { inject as service } from '@ember/service';
 
-export default class DashboardRoute extends Route.extend(AuthenticatedRouteMixin) {
-  authenticationRoute = 'login-error';
-  @service store;
-  @service currentUser;
+export default class IndexRoute extends Route {
+  @service session;
+  @service router;
 
-  async model() {
-    return hash({
-      schools: this.store.findAll('school'),
-      academicYears: this.store.findAll('academic-year'),
+  beforeModel(transition) {
+    this.session.requireAuthentication(transition, 'login-error');
+    this.router.replaceWith('dashboard', {
+      queryParams: {
```

Comment:
I don't think all of these params are needed for the default WaaG dashboard are they?

Reply:
:heavy_check_mark: